gatorgrade/input/command_line_generator.py

```diff
@@ -28,19 +25,19 @@ def generate_checks(file_context_checks):
         # If the check has a 'command', then it is a shell check
         if "command" in check:
             shell_checks.append(check)
-        # Else it's a gator grader check
+        # Else it's a GatorGrader check
         else:
             gatorgrader_command_options = []
             # Defining the description and option
-            description = check["description"]
-            options = check["options"]
-            # Creating a list that has description, check, and options in it for GatorGrader check
-            gatorgrader_command_options = ["--description", f"{description}"]
+            description = check.get("description")
+            options = check.get("options")
```

Comment:
Good idea to change to using `dict::get` here!

Reply:
Thanks!